modules/bench/src/bench.js

```diff
@@ -138,56 +146,61 @@ function runCalibrationTests({tests}) {
   }
 }
 
+function logEntry(test, opts) {
+  const priority = (global.probe && global.probe.priority) | 10;
+  if ((opts.priority | 0) <= priority) {
+    test.opts.log(opts);
+  }
+}
+
 // Run a list of bench test case async
-function runAsyncTests({tests, onBenchmarkComplete = noop}) {
+function runTestsAsynchronously({tests, onBenchmarkComplete = noop}) {
```

Comment:
Nit: usually these would be named `runTests` and `runTestsSync`

Reply:
> Nit: usually these would be named `runTests` and `runTestsSync`

OK dropped `Asynchronously`. 

This was not intended to offer such an interface distinction, more an attempt to create a more self-documenting name for the function, as the old name was rather confusing (at least in context).